docs/pages/versions/unversioned/sdk/notifications.mdx

```diff
@@ -186,11 +187,72 @@ Notifications.scheduleNotificationAsync({
 });
 ```
 
-### Handle push notifications with React Navigation
+### Handle push notifications with navigation
 
-If you'd like to deep link to a specific screen in your app when you receive a push notification, you can configure React Navigation's [linking](https://reactnavigation.org/docs/navigation-container#linking) prop to do that:
+If you'd like to deep link to a specific screen in your app when you receive a push notification, you can configure either of Expo's navigation systems to do that.
 
-```tsx
+<Tabs>
+
+<Tab label="Expo Router">
+
+You can use Expo Router's [built-in deep linking](/routing/create-pages) to handle incoming URLs from push notifications. Simply configure the root layout to listen for incoming and initial notification events.
+
+```tsx app/_layout.tsx
+import React from 'react';
```

Comment:
```suggestion
import { useEffect } from 'react';
```

Small nit for consistency since with latest SDKs, we started importing hooks directly from React library in code examples.